src/common/transformations/src/transformations/common_optimizations/fuse_rotary_positional_embeddings.cpp

```diff
@@ -652,8 +653,8 @@ ov::pass::RoPEFusionQwen::RoPEFusionQwen(int split_output_id) {
     auto Multiply_567524 = makePattern<opset1::Multiply>({ShapeOf_485735, {-1}}, {{"auto_broadcast", "numpy"}});
     auto Gather_377635 = makePattern<opset8::Gather>({Multiply_567524, {1}, 0}, {{"batch_dims", 0}});
 
-    auto input_ids = makePattern("i32[?,?]");  // [batch, length]
-    auto ShapeOf_409241 = makePattern<opset1::ShapeOf>({input_ids}, {});
+    auto input_ids = makePattern();  // [batch, length]
```

Comment:
Why did you remove `"i32[?,?]"` ?

Reply:
Because for some reason this complication leads to the detection of only 2/64 cases and it is easier to simplify the check than to add several more conditions